README.md

```diff
@@ -223,32 +246,39 @@ Depending on the `type` the structure of the `dataObject` differs.
     id: 'E2784NQXTHEN',
     from: '2017-12-05',
     to: '2017-12-16',
-    editUrl: 'url://for.editting/?the=item',
-    availabilityUrl: 'url://to-do.an/availability/check',
     price: 860,
     currencyCode: 'EUR',
   },
   trip: {
     destination: 'YYZ',
     alias: 'Die Küste Südkaliforniens (ab San Francisco)',
-    imageUrl: 'url://to-vehicle.img',
+    imageUrl: 'url://to-round-trip.img',
```

Comment:
You could use the `example` scheme if you want to be super standard conformant: https://tools.ietf.org/html/rfc7595#section-8